tensorflow/lite/micro/kernels/arc_mli/pooling.cc

```diff
@@ -62,13 +75,75 @@ TfLiteStatus CalculateOpData(const TfLiteContext* context,
       /*dilation_rate_height=*/1,
       /*dilation_rate_width=*/1, height, width, params->filter_height,
       params->filter_width, params->padding, &out_height, &out_width);
+  return kTfLiteOk;
+}
+
+void* Init(TfLiteContext* context, const char* buffer, size_t length) {
+  TFLITE_DCHECK(context->AllocatePersistentBuffer != nullptr);
+  return context->AllocatePersistentBuffer(context, sizeof(OpData));
+}
+
+TfLiteStatus Prepare(TfLiteContext* context, TfLiteNode* node) {
+  TFLITE_DCHECK(node->builtin_data != nullptr);
+  auto* params = reinterpret_cast<TfLitePoolParams*>(node->builtin_data);
```

Comment:
static_cast?

Reply:
https://github.com/tensorflow/tensorflow/blob/7978215e252e786f152504ab8999879aa6850cc6/tensorflow/lite/micro/kernels/pooling.cc#L217-L219

I just made it like it is in the reference kernel, what do you think?